scss/_responsive.scss

```diff
@@ -0,0 +1,11 @@
+@media (max-width: $mobile-width) {
```

Comment:
Dang, the latest commit lost this line's comments. Did @yegor256 you get my response?